app/views/dashboard/_stripe_account_setup_banner.njk

```diff
@@ -0,0 +1,31 @@
+{% if stripeAccountSetup and (not stripeAccountSetup.bankAccount or not stripeAccountSetup.vatNumberCompanyNumber or not stripeAccountSetup.responsiblePerson) %}
```

Comment:
Do we need to ensure we only show to admin users in this PR or will that come in a further PR?

Reply:
Good spot! I am going raise a separate PR for this